app/tests/utils/componentTestingHelper.tsx

```diff
@@ -76,10 +77,16 @@ class ComponentTestingHelper<
   };
 
   public renderComponent(
+    useDefaultDateSettings: boolean,
     getPropsFromTestQuery: (data: TQuery['response']) => any = this.options
       .getPropsFromTestQuery,
     extraProps: any = this.options.defaultComponentProps
+    
```

Comment:
```suggestion
    useDefaultDateSettings: boolean,
    getPropsFromTestQuery: (data: TQuery['response']) => any = this.options
      .getPropsFromTestQuery,
    extraProps: any = this.options.defaultComponentProps
    useDefaultDateSettings = true
```
Setting it to `true` by default would make all of our test use the default zone and local, which is what we want. Not sure if this actually needs to be a parameter, do we have test cases that need a different default zone or locale?

Reply:
We have in tests: 
```   const mockCurrentTime = DateTime.utc(2020, 1, 1, 5, 0);
    Settings.now = () => mockCurrentTime.toMillis();
    Settings.defaultZone = "UTC";

    componentTestingHelper.loadQuery();
    componentTestingHelper.renderComponent(false); ```

Reply:
Is there a reason why this specific test uses UTC, does it work if we set to the pacific time zone instead? I may have written this test and I think this was an arbitrary choice

Reply:
ok, let me try to change from DateTime.utc(2020, 1, 2, 0, 0); to DateTime.local(2020, 1, 2, 0, { zone: "America/Vancouver" })

Reply:
Fixed